src/Init/Init_ResetParameter.cpp

```diff
@@ -643,6 +643,45 @@ void Init_ResetParameter()
 #  endif
 
 
+// set default value for OPT__RECORD_CENTER
+   if ( OPT__RECORD_CENTER )
+   {
+      if ( COM_CEN_X < 0.0  ||  COM_CEN_Y < 0.0  ||  COM_CEN_Z < 0.0 )
+      {
+         COM_CEN_X = -1.0;
+         COM_CEN_Y = -1.0;
+         COM_CEN_Z = -1.0;
+         PRINT_RESET_PARA( COM_CEN_X, FORMAT_REAL, "and it will be reset to the coordinate of the peak total density" );
+         PRINT_RESET_PARA( COM_CEN_Y, FORMAT_REAL, "and it will be reset to the coordinate of the peak total density" );
+         PRINT_RESET_PARA( COM_CEN_Z, FORMAT_REAL, "and it will be reset to the coordinate of the peak total density" );
+      }
+
+      if ( COM_MAX_R < 0.0 )
+      {
+         COM_MAX_R = HUGE_NUMBER;
+         PRINT_RESET_PARA( COM_MAX_R, FORMAT_REAL, "by default" );
+      }
+
+      if ( COM_MIN_RHO < 0.0 )
+      {
+         COM_MIN_RHO = 0.0;
+         PRINT_RESET_PARA( COM_MIN_RHO, FORMAT_REAL, "by default" );
+      }
+
+      if ( COM_TOL_ERR_R < 0.0 )
+      {
+         COM_TOL_ERR_R = 0.1*COM_MAX_R;
+         PRINT_RESET_PARA( COM_TOL_ERR_R, FORMAT_REAL, "by default" );
+      }
+
+      if ( COM_N_ITER_MAX < 0 )
+      {
+         COM_N_ITER_MAX = 10;
```

Comment:
Just curious why not set the default value to `10` directly in `Input__Parameters` and `Init_Load_Parameter()`.